### DISCUSSION TITLE
Newbie question

### DISCUSSION THREAD

**Discussion by owangen** [orta/vscode-jest#765] [Q&A] [answered]
Hi

I'm contributing to a open source project that uses Jest for test (https://github.com/Koenkk/zigbee-herdsman-converters). My IDE is vscode on Windows (latest version). I have installed jest, but have som problems running Jest on this project. In package.json the script section says `"test": "ZHC_TEST=true jest test"`, and when running the test it complains about the argument `"ZHC_TEST=true"`. Any ideas how to proceed?

**Comment by rster2002** [comment, marked as answer]
Not sure if you still need help as it has been a while, but I'll answer anyways.

Windows doesn't support setting environment variables that way. Your best bet is to use something WSL if you have it as this will use linux to run the script. If you don't have WSL you can take a look at [cross-env](https://www.npmjs.com/package/cross-env) and change the script to be: `"test": "cross-env ZHC_TEST=true jest test"` Hope this helps you in any way 👍

**Reply by owangen** [reply]
Thanks a lot. Yes, I actually have WSL installed, but is not using it for anything. This is a good reason to start using it. And thanks for the tip about cross-env setting, but I guess it's better to use WSL because then I don't need to change anything in the code.
